pilot/pkg/features/pilot.go

```diff
@@ -422,6 +432,9 @@ var (
 		"If enabled, addition runtime asserts will be performed. "+
 			"These checks are both expensive and panic on failure. As a result, this should be used only for testing.",
 	).Get()
+
+	TLSInboundCipherSuites = env.RegisterStringVar("TLS_INBOUND_CIPHER_SUITES", DefaultInboundCiphers,
```

Comment:
Input validation?

We should parse what is passed, log WARN for invalid CIPHERS and retain what is valid. If nothing valid is passed we should ensure a functioning default or exit.

Bonus - reject legacy supported but compromised ciphers. Probably best to pick a functional subset of TLS1.2 & TLS1.3

@PiotrSikora

Reply:
Added an input validation mechanism that checks whether each cipher is on the allowed cipher list (the allowed cipher list is based on the Envoy BoringSSL FIPS cipher list on
https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto.html).
If nothing valid is passed, the default cipher suites are used.